src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -529,10 +529,9 @@ export default class SourceFileLinter {
 			if (isDataType) {
 				this.#reporter.addMessage({
 					node: moduleSpecifierNode,
-					severity: LintMessageSeverity.Error,
-					ruleId: RULES["ui5-linter-no-pseudo-modules"],
-					message: formatMessage(MESSAGES.SHORT__NO_DIRECT_DATATYPE_ACCESS, moduleSpecifierNode.text),
-					messageDetails: formatMessage(MESSAGES.DETAILS__NO_DIRECT_DATATYPE_ACCESS, moduleNamespaceName),
+					message: MESSAGE.NO_DIRECT_DATATYPE_ACCESS,
```

Comment:
Why changing the signiture as we already have the required props grouped in here.
Maybe, instead we can directly expand?
```suggestion
					...MESSAGE.NO_DIRECT_DATATYPE_ACCESS,
```

Reply:
I think that could be done (but then based on the MESSAGE_INFO). 
owever, it would most probably not allow to reduce memory usage (see my comment https://github.com/SAP/ui5-linter/pull/104#discussion_r1706802866), as objects/strings are already created.
Also, it would again allow providing the properties individually, which is IMHO not necessary and error-prone.